src/main.ts

```diff
@@ -295,6 +321,24 @@ function handleKeyDown({ key }: KeyboardEvent): void {
   }
 }
 
+async function handleFindTrack(): Promise<void> {
+  // Load track value
+  const trackId = trackInput.valueAsNumber;
+  const newTrack = dataset!.buildTrack(trackId);
+
+  // Check for track validity
+  if (newTrack.length() > 0) {
```

Comment:
This is an arbitrary style preference, but since this is validation I might flip the condition and make it an early return, e.g.
```typescript
if (newTrack.length() < 1) {
  return;
}
```
otherwise if this function had multiple layers of validation, the code would start drifting to the right.